activity/activity.info

```diff
@@ -6,3 +6,7 @@ icon = activity-blockparty
 activity_version = 8
 license = MIT
 show_launcher = yes
+summary = https://github.com/sugarlabs/block-party-activity 
```

Comment:
@Gabbar8055 Thanks for the patch.
The summary as described [here](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/bundle/__init__.py) should give a basic description of the activity. Please fill the appropriate metadata.

Reply:
Done!